README.md

```diff
@@ -99,6 +100,15 @@ You can have the tool create a machine-readable log file by specifying the `--lo
                      the configuration of your cf instance.
 ```
 
+##### dump [SUBCOMMAND]
+
+```
+  all               Resolve $ref-occurrences in an entire configuration file and print the result.
```

Comment:
IMO a complete `dump` would've been sufficient.

Reply:
```suggestion
  all               Read complete configuration, resolve all $refs and print the result to the console.
```

I think this entire description is misleading, though. What's an "entire" or "complete" configuration? Why are we still introducing new code about the soon-to-be-removed services/applications/...?

I really think it just creates complexity, and is likely going to be removed soon enough again.

Reply:
Yes, by only supporting a single dump command, we could easily remove complexity. I will ask the POs in today's meeting about that.

Reply:
It would be good, to see the complexity/size of a yaml file from real user.
If the size and complexity of a yaml may be really big, I think for some users those subcommands would be great.

Reply:
The POs decided to leave them in for now. In a later sprint we might remove the subcommands again.

Reply:
After talking to the industry partner, the POs now decided that we should no longer support the sum-commands.